### PR TITLE
Add support for Debian(jessie) package

### PR DESCRIPTION
**Pull request by kalistace**

### CODE REVIEW

**Comment by jsullivan3** [comment]
This change looks really cool.  I'll give it a sanity test, though I'll admit I'm more familiar with RPM-based distributions than Debian-derived ones.  Thank you very much for yet another contribution!

**Comment by jsullivan3** [comment]
I was able to configure grunt and create a Debian package on Jessie using these instructions, though I did need to configure a few things in order to build the package:
```
$ sudo apt-get install -y npm nodejs-legacy devscripts debhelper
$ sudo npm install -g grunt-cli
$ npm install
$ grunt debian_package
```
Since I'm rather new to Debian and grunt, I wanted to check if you think that the full list of steps would be good to add, or if the packages required to make ```grunt debian_package``` succeed would be more obvious to a Debian user.

Other than a possible README update to include the required packages, this change looks great to me.  Please let me know if you think a README update is necessary, and I'll merge this PR once we come to a consensus.

Thanks!

**Comment by kalistace** [comment]
Hi,

Thanks for your time! 

To be honest, I'm not really familiar with Grunt either. I just assumed it was a commonly used tool in the nodejs environment.

Anyway, I've added additional documentation just in case :)

Let me know if it makes things a bit clearer.

Thanks !

File: readme.md [hunk 1]
```diff
@@ -313,6 +313,48 @@ Note :
...
+
+git2consul can be packaged in .deb file. Simply run
+ 
+ ```
+ npm install
+ grunt debian_package
+ ```
+
+
+This task use the [grunt-debian-package](https://www.npmjs.com/package/grunt-debian-package) and depends on two debian tools :
```
Comment (jsullivan3) [inline readme.md]: Very minor grammatical nit:  This task _**uses**_ the...

**Comment by kalistace** [comment]
Yeap sorry about that :}

**Comment by jsullivan3** [comment]
That was fast - thanks!  Merging now - thanks again for your contribution.

**Pull request merged**